src/backend/distributed/executor/placement_access.c

```diff
@@ -128,8 +128,8 @@ BuildPlacementAccessList(int32 groupId, List *relationShardList,
 	RelationShard *relationShard = NULL;
 	foreach_ptr(relationShard, relationShardList)
 	{
-		ShardPlacement *placement = FindShardPlacementOnGroup(groupId,
-															  relationShard->shardId);
+		ShardPlacement *placement = ActiveShardPlacementOnGroup(groupId,
```

Comment:
I'm not sure how to test this change. @onderkalaci @marcocitus do you have any pointers?

Reply:
I think we cannot reach this code for not active placements, so no.